prometheus/README.md

```diff
@@ -0,0 +1,124 @@
+
+As an alternative, you can install the upstream chart with all customization options parallel. This tutorial outlines how to set up such installation in co-existence to the Kyma monitoring stack.
+
+> **CAUTION:** This tutorial describes a basic setup that you should not use in production. Typically, a production setup needs further configuration, like optimizing the amount of data to scrape and the required resource footprint of the installation. To achieve qualities like [high availability](https://prometheus.io/docs/introduction/faq/#can-prometheus-be-made-highly-available), [scalability](https://prometheus.io/docs/introduction/faq/#i-was-told-prometheus-doesnt-scale), or [durable long-term storage](https://prometheus.io/docs/operating/integrations/#remote-endpoints-and-storage), you need a more advanced setup.
+
+> **CAUTION:** This example will use the latest Grafana version which is under AGPL-3.0 and might not be free of charge for commercial usage.
+
+## Prerequisites
+
+- Kyma as the target deployment environment.
+- Kubectl > 1.22.x
+- Helm 3.x
+
+## Installation
+
+### Preparation
+
+1. Export your Namespace as a variable. Replace the `{namespace}` placeholder in the following command and run it:
+
+    ```bash
+    export KYMA_NS="{namespace}"
+    ```
+1. If you don't have it created yet, now is the time to do so:
+    ```bash
+    kubectl create namespace $KYMA_NS
+    ```
+
+1. Assure that your namespace has istio-injection disabled by having the proper label in place
+    ```bash
+    kubectl label namespace $KYMA_NS istio-injection=disabled
+    ```
+    >**Note**: This Namespace must have **no** Istio sidecar injection enabled. The Helm chart deploys jobs that will not succeed when sidecar injection is enabled by default.
+
+1. Export the Helm release name that you want to use. It can be any name, but be aware that all resources in the cluster will be prefixed with that name. Replace the `{release-name}` placeholder in the following command and run it:
+   ```bash
+    export HELM_RELEASE="{release-name}"
+    ```
+
+1. Update your Helm installation with the required Helm repository:
+
+    ```bash
+    helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
+    helm repo update
+    ```
+
+### Install the kube-prometheus-stack
+
+1. Run the Helm upgrade command, which installs the chart if it's not present yet. At the end of the command, change the Grafana admin password to some value of your choice.
+    ```bash
+    helm upgrade --install -n ${KYMA_NS} ${HELM_RELEASE} prometheus-community/kube-prometheus-stack -f https://raw.githubusercontent.com/kyma-project/examples/main/prometheus/values.yaml --set grafana.adminPassword=myPwd
```

Comment:
Default kyma setup does not have a `--denyNamespaces` setup. see [here](https://github.com/kyma-project/kyma/blob/5ea7dbd64a1bf4b21b7198983cfe5c08599919b8/resources/monitoring/values.yaml#L1745). So the monitoring operator coming from the Kyma would interfere with the operator installed by above command and would cause constant restarts of the various pods.

Reply:
fixed by one more prep step